src/Appwrite/SMS/Adapter/Dialog360.php

```diff
@@ -0,0 +1,59 @@
+<?php
+
+namespace Appwrite\SMS\Adapter;
+
+use Appwrite\SMS\Adapter;
+
+// Reference Material
+// https://docs.360dialog.com/docs/whatsapp-api/whatsapp-api/sandbox#php
+
+class Dialog360 extends Adapter
+{
+    /**
+     * @var string
+     */
+    private string $endpoint = 'https://waba.360dialog.io/v1/messages';
```

Comment:
Can you actually make 2 Adapters: 1 called Dialog360 and the other called Dialog360Sandbox. The difference would be the endpoint? I think it makes sense for 1 to be a subclass of the other.